### ISSUE TITLE
Update openapi specs with TxStatus changes

### ISSUE THREAD

**Issue by ninabarbakadze** [celestiaorg/celestia-core#1406]
utAck.

Awesome stuff 🔥  Left some unblocking nits.

Also, would be good to update the openapi specs to reflect these changes both in main in this branch. Can be handled in a separate PR or an issue for this to be done later.

_Originally posted by @rach-id in https://github.com/celestiaorg/celestia-core/pull/1405#pullrequestreview-2151255035_

**Comment by evan-forbes** [comment]
can we close this as completed or is this still something we need to finish?
